feat: add docs for setting up a trading post reverse proxy (#37)
useverto/trading-post: README.md

You've successfully started a verto trading post! :smile:

> You can also [setup a reverse proxy](./docs/PROXY.md) with `nginx` for the trading post

Reviewer: ```suggestion
You can also [set up a reverse proxy](./docs/PROXY.md) with `nginx`.
```

#### Conclusion

> After you've started the trading post, you might want to [setup a reverse proxy](./docs/PROXY.md) for the trading post API!

Reviewer: ```suggestion
After you've started the trading post, you might want to [set up a reverse proxy](./docs/PROXY.md) for the trading post API.
```
